crates/infra/agentic-config/CLAUDE.md

```
@@ -52,7 +53,13 @@ Model selection for `ask_reasoning_model` tool:
 - `reasoning_effort`: Optional effort level (`low`, `medium`, `high`, `xhigh`)
 
 ### `services` - External APIs
-Base URLs for Anthropic and Exa APIs. API keys are loaded from environment only (never serialized).
+Base URLs and timeout knobs for Anthropic, Exa, Linear, and GitHub integrations. API keys are loaded from environment only (never serialized).
+
+### `review`
```

markdownlint-cli2 (0.22.1)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)